src/policy/fees.cpp

```diff
@@ -390,8 +390,9 @@ void CBlockPolicyEstimator::processTransaction(const CTxMemPoolEntry& entry, boo
         mapMemPoolTxs[hash].bucketIndex = feeStats.NewTx(txHeight, (double)feeRate.GetFeePerK());
     }
     else {
-        LogPrint("estimatefee", "not adding\n");
+        LogPrint("estimatefee", "not adding");
     }
+    LogPrint("estimatefee", "\n");
```

Comment:
It might be ugly, but it was working correctly before right?  This will cause extra new lines I think, because NewTx above prints a new line.  I guess we could just remove it from there.  This may be a case of too much debugging, I'm not sure if we want all these anyway....